cf-custom-resources/test/custom-domain-app-runner-test.js

```diff
@@ -27,643 +27,1189 @@ describe("Custom Domain for App Runner Service During Create", () => {
                 setTimeout(resolve, 1000);
             });
         }); // Mock deadline should time out after normal operations.
+
+        console.log = function () {};
```

Comment:
Oh I added it in my previous PR https://github.com/aws/copilot-cli/pull/2589/files#diff-19dda72588eb5ea69bf48f9b0a8613d43aec245816c8f11c549fc771d4630c27R21